lms/djangoapps/teams/csv.py

```diff
@@ -406,21 +432,31 @@ def is_FERPA_bubble_breached(self, teamset_id, team_name):
         else:
             return True
 
-    def validate_proposed_team_size_wont_exceed_maximum(self, team_name, teamset_id):
+    def validate_team_sizes_not_exceeded(self):
         """
         Validates that the number of users we want to add to a team won't exceed maximum team size.
         """
-        if self.course.teams_configuration.teamsets_by_id[teamset_id].max_team_size is None:
-            max_team_size = self.course.teams_configuration.default_max_team_size
-        else:
-            max_team_size = self.course.teams_configuration.teamsets_by_id[teamset_id].max_team_size
+        for teamset_id in self.teamset_ids:
+            # Get max size for team-set
+            if self.course.teams_configuration.teamsets_by_id[teamset_id].max_team_size is None:
+                max_team_size = self.course.teams_configuration.default_max_team_size
+            else:
+                max_team_size = self.course.teams_configuration.teamsets_by_id[teamset_id].max_team_size
+
+            if max_team_size is not None:
```

Comment:
How would max_team_size be None?

Reply:
Good catch, I think that was left over from an earlier refactor. Removed.